website/docs/guides/middlewares/batch-consume-middleware.md

```diff
@@ -43,6 +43,10 @@ services.AddKafka(kafka => kafka
 
 To access the batch from the next middleware, use the `GetMessagesBatch` method accessible through the `context` argument.
 
+:::warning
```

Comment:
Those this work? 🤔
I can't find `warning` in the list [here](https://docusaurus.io/docs/next/markdown-features/admonitions)

Reply:
From what I see it does, we already use it on the consumers.md file, it seems to do the same as :::danger